src/test/rustdoc-gui/search-input-mobile.goml

```diff
@@ -0,0 +1,14 @@
+// Test to ensure that you can click on the search input, whatever the width.
+// The PR which fixed it is: https://github.com/rust-lang/rust/pull/81592
+goto: file://|DOC_PATH|/index.html
+size: (463, 700)
```

Comment:
This controls the size of the user's browser?

Reply:
Exactly, doc for this command is here: https://github.com/GuillaumeGomez/browser-UI-test/#size